modules/jdbc/src/integrationTest/java/org/apache/ignite/jdbc/ItJdbcMultiStatementSelfTest.java

```diff
@@ -17,21 +17,25 @@
 
 package org.apache.ignite.jdbc;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 /**
- * Tests for ddl queries that contain multiply sql statements, separated by ";".
+ * Tests for queries containing multiple sql statements, separated by ";".
```

Comment:
It seems to me that we can unmute the `testEmptyStatements` test?

I also haven't found a test that checks the following scenario.
```
START TRANSASCTION; // RW TX
SELECT * FROM TABLE(system_range(0, 2000))  // tab size must be greater than a "fetch size"
COMMIT; // blocks until previous cursor is closed
START TRANSASCTION;
SELECT 2; 
COMMIT;
```
the user should be able to switch to SELECT statement 2 without explicit close/read the first select statement